pkg/traceql/test_examples.yaml

```diff
@@ -51,6 +51,14 @@ valid:
   - '{ duration > 1s }'
   - '{ 1 < 1h }'
   - '{ 1 <= 1.1 }'
+  # scoped intrinsics
```

Comment:
i'd add a few examples under "validate_fails" to make sure that we can't compare the new intrinsics in incorrect ways.

maybe the other sections?